### DISCUSSION TITLE
Resources for design principles to be followed for Block development in Woo extensions

### DISCUSSION THREAD

**Discussion by Sidsector9** [woocommerce/woocommerce#42709] [General]
Hey all 👋  While building Blocks for Woo extensions, are there any set of design guidelines to be followed? At the moment, I'm selecting UI components that take the least time to develop, which I feel may not the best ones. Any good tips/resources?

**Comment by shaybanshee** [comment]
Hey there! We do have [UX guideline](https://github.com/woocommerce/woocommerce/tree/trunk/docs/user-experience) docs which are evolving as we add them to our docs repo, as well as best practices laid out for [blocks design](https://developer.wordpress.org/block-editor/explanations/user-interface/block-design/) in WP. Hope this helps!
